projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -31,8 +31,9 @@ export class TabParagraphContainerComponent
   @ViewChildren(ComponentWrapperDirective)
   children!: QueryList<ComponentWrapperDirective>;
 
-  tabTitleParams: Observable<any>[] = [];
+  tabTitleParams: (Observable<any> | undefined)[] = [];
 
+  // TODO: it is not used any more, so can be removed in 5.0
```

Comment:
Perhaps create a followup ticket with labels `order-lib` and `Next Major`

Reply:
https://github.com/SAP/spartacus/issues/13644 is created